sections/preliminaries.tex

```diff
@@ -0,0 +1,136 @@
+\Gls{lda} uncovers this semantic structure by discovering patterns of word use in documents and finding topics based on these~\cite{blei2009topic}.
+
+The standard \gls{lda} by \citet{blei2003latent} can be described by the following generative process, which is the way the model assumes the documents arose:
+$D$ is the number of documents in the corpus, $N_d$ is the number of words in document $d$, $V$ is the size of the vocabulary, and $K$ is the number of topics.
+Topics are represented as distributions over words and documents are represented as distributions of topics.
+\Gls{lda} assumes that the topics are shared across the corpus, while the document-topic distributions are unique for each document.
+For each topic $k \in \{1,\dots, K\}$ a topic-word distribution $\beta_k$ is sampled from a V-dimensional Dirichlet distribution parameterized by $\eta$.
+That is, K topics $\beta_{1:k}$ are sampled, each being a distribution over the vocabulary, written as: $\beta_k \sim Dirichlet(\eta)$.
+Likewise, for each document $d \in \{1,\dots, D\}$ a document-topic distribution $\theta_d$ is sampled from a K-dimensional Dirichlet distribution parameterized by $\alpha$.
+For each word $n \in \{1, \dots, N_d\}$ in each document $d$, a topic $z_{d,n}$ is sampled from a K-multinomial distribution $\theta_d$, and then a word $w_{d,n}$ is sampled from a V-multinomial distribution $\beta_{z_{d,n}}$.
+The generative process for each document is seen in these steps:
+
+\vspace{\topsep}
+\begin{enumerate}
+	\item Draw topic proportion $\theta_d \sim Dirichlet(\alpha)$
+	\item For each word $n$ in the document:
+	\begin{enumerate}
+		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_d)$
+		\item Draw word $w_{d,n} \sim Mult(\beta_{z_{d,n}})$
+	\end{enumerate}
+\end{enumerate}
+\vspace{\topsep}
+
+The generative process for topics and documents, generates a list of $K$ topics and $D$ documents that can be used as a $K \times V$ matrix of topic-word distributions and a $D \times K$ matrix of document-topic distributions, respectively.
+The plate notation for \gls{lda} can be seen in \autoref{fig:standard_lda}.
+
+\input{figures/lda_plate}
+
+After training the \gls{lda} model, there are multiple possibilities for exploring the corpus using the posterior distributions of the hidden random variables.
+One possibility is to visualize the posterior topics of the model, e.g., by sorting $\beta_k$ according to the highest probabilities. \vejleder{gerne mere præcist}
+It is also possible to visualize the documents by, e.g., sorting by the highest topic proportions in $\theta_d$.
+Another possibility of exploration is finding similar documents by using a distribution distance function on the topic proportions $\theta_d$ between documents~\cite{blei2009topic}.
+
+
+\subsection{Author-Topic \gls{lda}}\label{subsec:auth_prelim} \vejleder{introducer modellen som den er i \cite{author_topic_2012}, gerne flere detaljer}
```

Comment:
this section is very small. Maybe we should also go over generative process, etc. unless we want it to be small

Reply:
\todo[inline]{Beskriv den generative process}